### ISSUE TITLE
:sparkles: Video Versions Indicator

### ISSUE THREAD

**Issue by Dml2023** [jurialmunkey/skin.arctic.fuse#394]
### Skin section

Library

### Feature description

Hi JurialMunkey,

Was wondering if you had planned on adding a custom indicator for Video Versions like you already have in place for Library, Collections, etc.

### Why is this feature needed? Please justify and explain.

This would allow a quick visual indication when browsing the library or on a widget similar to what the Estuary skin has implemented.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Yeah definitely. I've actually got a few different things planned for the versions feature -- just waiting on a couple of PRs for versions to get merged into core.

I'm really keen to use it to display a codec style flag too for some of the more common version types -- e.g. to show a Blu-Ray icon for Blu-Ray version etc. -- going to be way better than the old methods which relied on filenames.

**Comment by Dml2023** [comment]
That sounds incredible. I can't wait to see what you implement.
